post_installation_configuration/aws-compute-edge-tasks.adoc

```diff
@@ -0,0 +1,98 @@
+:_content-type: ASSEMBLY
+:context: aws-compute-edge-tasks
+[id="aws-compute-edge-tasks"]
+= AWS Local Zone tasks
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+After installing {product-title} in AWS, you can further expand and customize your cluster
+to your requirements with support of Local Zones and edge compute pool.
+
+[id="extend_existing_to_local_zones_day2"]
+== Extend existing clusters to use AWS Local Zones
+
+You can extend an existing {product-title} installation in AWS to use AWS Local Zones for Day 2 operations.
+
+To extend nodes to Local Zone locations, you must create a subnet in the VPC for the target zone,
+adjust the cluster network MTU, and create the Machine set manifest to create the node
+into that location.
+
+[role="_additional-resources"]
+.Additional resources
+
+* See link:https://aws.amazon.com/about-aws/global-infrastructure/localzones/features/[AWS Local Zones features] in the AWS documentation for more information about AWS Local Zones and the supported instances types and services.
```

Comment:
I think it is safe. We are doing it in another pieces, including from where I used as reference: https://docs.openshift.com/container-platform/4.13/post_installation_configuration/cluster-tasks.html#installation-extend-edge-nodes-aws-local-zones_post-install-cluster-tasks

Reply:
I cannot disagree with that link. Hopefully AWS won't change the link URL and a customer notices the discrepancy. The docs team do not periodically check external links that are added to docs.

Reply:
Yes! But let me know if you have any other suggestions about it, because they provides the product and we the automation. So we need to document that integration, the product is the provider’s responsibility.

Reply:
I do no think there is anything we can do. If we document their product according to their integration capabilities, we could be open to liability issues if we don't keep pace with their updates. Feel free to resolve this thread.